deep_qa/data/token_indexers/token_indexer.py

```diff
@@ -75,7 +74,6 @@ def pad_token_sequence(self,
         """
         raise NotImplementedError
 
-    @staticmethod
-    def from_params(params: Params):
-        choice = params.pop_choice('type', list(token_indexers.keys()), default_to_first_choice=True)
-        return token_indexers[choice].from_params(params)
```

Comment:
Just move the import inside the method to remove the circular import.  We really do want everything to be _constructed_ `from_params`, so that we can have a single configuration file that runs a whole experiment.  Having this here makes it so that things like [this](https://github.com/allenai/deep_qa/blob/86d280c7eab491aedbc7fb38a4e1311ee4c31c1d/deep_qa/data/dataset_readers/snli.py#L69) work.  Otherwise it's more complicated to do the construction.